### ISSUE TITLE
Add a new parameter for vkrelpay to choose wsi.

### ISSUE THREAD

**Issue by jyangsh** [LunarG/VulkanTools#433]
Some platforms may support multi-wsi, e.g. xcb, display, etc. Can we add a new option for vkreplay to choose the wsi?

**Comment by joey-lunarg** [comment]
Adding WSI selection to vkreplay as a command-line argument is on the roadmap.

In the meantime, you can set the cmake build option VKREPLAY_WSI_SELECTION to either XCB or WAYLAND.

**Comment by jyangsh** [comment]
Yes, I know the build option, but it need re-build or using different version binaries. Sometimes, it is annoying. :)
